dashboard/app/models/ability.rb

```diff
@@ -97,6 +99,10 @@ def initialize(user)
             district.contact_id == user.id
           end
         end
+        can :group_view, Plc::UserCourseEnrollment
+        can :manager_view, Plc::UserCourseEnrollment do |enrollment|
+          DistrictsUsers.exists?(user: enrollment.user, district: District.where(contact: user.id))
```

Comment:
Super small nit: you can pluck the district ID to avoid loading the whole object.
```ruby
irb(main):001:0> District.where(contact: user.id)
  District Load  SELECT `districts`.* FROM `districts` WHERE `districts`.`contact_id` = 1
irb(main):002:0> District.where(contact: user.id).pluck(:id)
  SELECT `districts`.`id` FROM `districts` WHERE `districts`.`contact_id` = 1
```